packages/shared/src/components/ExtensionPermissionsPrompt.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { ReactElement, useContext } from 'react';
+import Logo, { LogoPosition } from './Logo';
+import { OnboardingTitleGradient } from './onboarding/common';
+import { Button, ButtonSize } from './buttons/Button';
+import { cloudinary } from '../lib/image';
+import { ExtensionContext } from '../contexts/common';
+
+interface Props {
+  onSuccess?: () => void;
+}
+
+const ExtensionPermissionsPrompt = ({ onSuccess }: Props): ReactElement => {
+  const { requestHostPermissions, origins } = useContext(ExtensionContext);
+
+  const handleRequestHostPermissions = async () => {
+    const success = await requestHostPermissions({ origins });
+    if (success) {
+      onSuccess();
+    }
+  };
+
+  return (
+    <div className="flex overflow-hidden flex-col justify-center items-center px-7 antialiased text-center min-h-[100vh] max-h-[100vh]">
```

Comment:
Why don't we use `min-h-screen` here?
Also not sure why the px-7 is needed, mobile?

Reply:
I picked up the same classes that were used in the [ExtensionOnboarding](https://github.com/dailydotdev/apps/blob/1f19c3434f66f81d2e11a2c2ecdc94a3cb187301/packages/shared/src/components/ExtensionOnboarding.tsx#L11) component, I'll double check these and clean them up 👍

Reply:
Tested `min-h-screen` and it does not look like it works properly

Reply:
Let's leave it for now. perhaps we override something.